### ISSUE TITLE
In the 6th code block on the page $file seem to be unused

### ISSUE THREAD

**Issue by ybeltrikov** [MicrosoftDocs/PowerShell-Docs-Modules#197]
### Type of issue

Typo

### Feedback

In the code $file is defined but never used:
`[string]$file = $MyInvocation.MyCommand.Definition`

We possibly need to add it to the ArgumentList:
```
$objParams = @{
  TypeName = 'Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic.CorrectionExtent'
  ArgumentList = $startLineNumber, $endLineNumber, $startColumnNumber,
                 $endColumnNumber, $correction, $file, $optionalDescription
}
```

### Page URL

https://learn.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/create-custom-rule?view=ps-modules&source=docs

### Content source URL

https://github.com/MicrosoftDocs/PowerShell-Docs-Modules/blob/main/reference/docs-conceptual/PSScriptAnalyzer/create-custom-rule.md

### Author

@sdwheeler

### Document Id

ea102f70-2dbc-92a0-014f-967097599adf

**Comment by ybeltrikov** [comment]
Raised PR GH-198

**Comment by michaeltlombardi** [comment]
Resolved by #198
